packages/celotool/src/cmds/generate/faucet-load-test.ts

```diff
@@ -0,0 +1,95 @@
+/* tslint:disable no-console */
+import { newKit } from '@celo/contractkit'
+import { switchToClusterFromEnv } from 'src/lib/cluster'
+import { convertToContractDecimals } from 'src/lib/contract-utils'
+import { addCeloEnvMiddleware, CeloEnvArgv, envVar, fetchEnv } from 'src/lib/env-utils'
+import { AccountType, generateAddress } from 'src/lib/generate_utils'
+import { portForwardAnd } from 'src/lib/port_forward'
+import yargs from 'yargs'
+
+interface FaucetLoadTest extends CeloEnvArgv {
+  count_from: number
```

Comment:
I now notice we elsewhere use `replicas` for this, which I think is a better than `count` or `pod` (one I had thought of.  So I would change the flags and variables here to use `replica` or `replicas` as well.